benches/bench.rs

```diff
@@ -26,17 +26,13 @@ fn navigator_benchmark(c: &mut Criterion) {
     bench!(read_pressure());
     bench!(read_temperature());
 
-    bench!(read_all());
```

Comment:
read_all isn't the same as read_adc_all, was it removed?

Reply:
Fixed